src/Microsoft.Framework.Localization/ResourceManagerStringLocalizerFactory.cs

```diff
@@ -34,7 +34,7 @@ public class ResourceManagerStringLocalizerFactory : IStringLocalizerFactory
             _resourcesRelativePath = localizationOptions.Value.ResourcesPath ?? string.Empty;
             if (!string.IsNullOrEmpty(_resourcesRelativePath))
             {
-                _resourcesRelativePath = _resourcesRelativePath.Replace("/", ".") + ".";
+                _resourcesRelativePath = _resourcesRelativePath.Replace("/", ".").Replace('\\', '.') + ".";
```

Comment:
Why are we using the string overload and char overload on the same line? Also why not Path.DirectorySeparatorChar/Path.AltDirectorySeparatorChar? Is there a good reason to not use those?